before_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+test -f "$(which docker )" || { echo "Docker not installed"; exit 1; }
+test -f "$(which npm)" || { echo "NPM not installed"; exit 1; }
+test -f "$(which node)" || { echo "Node not installed";
+    test -f "$(which nodejs)" && echo "NodeJs is installed however!";
```

Comment:
This will be confusing to the user since node==nodejs. Please elaborate on the symlink fix.